programs/bpf_loader/src/syscalls.rs

```diff
@@ -1431,6 +1432,9 @@ impl<'a, 'b> SyscallObject<BpfError> for SyscallMemcpy<'a, 'b> {
         let use_fixed_nonoverlapping_check = invoke_context
             .feature_set
             .is_active(&fixed_memcpy_nonoverlapping_check::id());
+        let do_check_physical_overlapping = invoke_context
+            .feature_set
+            .is_active(&check_physical_overlapping::id());
 
         #[allow(clippy::collapsible_else_if)]
         if use_fixed_nonoverlapping_check {
```

Comment:
Shouldn't we check `src_ptr`, `dst_ptr` instead of `src_addr`, `dst_addr` here?

Reply:
I'm leaving the original check in place, the caller should also not pass in overlapping virtual regions